tests/core/numerics/interpolator/interpolator_test.py

```diff
@@ -43,5 +43,15 @@ def main():
 
 
 
+def interpolate_datasets(beforeTime, afterTime, interp_time, beforeData, afterData):
```

Comment:
I think `time_interpolate()` makes more sense (not to be confused with spatial interpolation)